core/src/main/scala/org/locationtech/rasterframes/functions/TileFunctions.scala

```diff
@@ -104,11 +104,21 @@ trait TileFunctions {
 
   /** Resample tile to different size based on scalar factor or tile whose dimension to match. Scalar less
    * than one will downsample tile; greater than one will upsample. Uses nearest-neighbor. */
-  def rf_resample[T: Numeric](tileCol: Column, factorValue: T) = Resample(tileCol, factorValue)
+  def rf_resample[T: Numeric](tileCol: Column, factorValue: T) = ResampleNearest(tileCol, factorValue)
```

Comment:
In SQL this is called `rf_resample_nearest` (and as the `nodeName` per suggestion above). Is the rationale for backwards compatibility?

Reply:
back compatibility with the 2 argument form, yes.